src/App.js

```diff
@@ -10,10 +13,41 @@ const tabs = [
 ];
 /* eslint-enable */
 
-const App = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-  </div>
-);
+class App extends Component {
+  state={
+    tabs: [...tabs],
+    selectedTab: 'tab-1',
+  }
+
+  onTabSelected = (id) => {
+    this.setState(prevSate => ({
+      selectedTab: id,
+    }), () => {
+    });
```

Comment:
Цей другий необов'язковий
```suggestion
    })
```